go.mod

```diff
@@ -1,48 +1,53 @@
 module github.com/IBM/ibm-block-csi-operator
 
-go 1.13
+go 1.15
 
 require (
-	github.com/go-logr/logr v0.1.0
-	github.com/go-openapi/spec v0.19.3
-	github.com/go-test/deep v1.0.4 // indirect
-	github.com/golang/protobuf v1.4.2
-	github.com/imdario/mergo v0.3.9
-	github.com/onsi/ginkgo v1.12.1
-	github.com/onsi/gomega v1.10.1
-	github.com/operator-framework/operator-sdk v0.18.1
+	github.com/go-logr/logr v0.4.0
+	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334 // indirect
+	github.com/imdario/mergo v0.3.12
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
 	github.com/pkg/errors v0.9.1
-	github.com/presslabs/controller-util v0.1.14-0.20190818132821-fc080f9430fa
-	github.com/spf13/pflag v1.0.5
-	google.golang.org/grpc v1.27.0
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20200121204235-bf4fb3bd569c
-	sigs.k8s.io/controller-runtime v0.6.2
+	github.com/presslabs/controller-util v0.2.6
+	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
+	k8s.io/api v0.21.2
+	k8s.io/apimachinery v0.21.2
+	k8s.io/client-go v0.21.2
+	sigs.k8s.io/controller-runtime v0.9.2
 	sigs.k8s.io/yaml v1.2.0
 )
 
-// Pinned to kubernetes-1.18.0
+// Pinned to kubernetes-1.20.2
```

Comment:
shouldn't we align to k8s 1.21?

Reply:
this is how it generated
but I can try to use the latest one if you think it's neccecry

Reply:
is 1.21 the latest? how do we control it?

Reply:
yes
with `replace` in the go.mod

Reply:
I see.
then whatever you think is best (I suggested it because 1.7.0 will not support 1.20)

Reply:
done